src/poker.js

```diff
@@ -1,10 +1,69 @@
-function winningPair() {}
+const cardNumbers = {
+  J: '11',
+  Q: '12',
+  K: '13',
+  A: '14'
+}
+
+function pairCheck(pair) {
+  return pair[0] === pair[1]
+}
+
+function setCheck(hand) {
+  return hand.length === 3 && hand[0] === hand[1] && hand[1] === hand[2]
+}
+
+const checkNumber = (pairs) =>
+  pairs.map((pair) => Number(cardNumbers[pair] || pair))
+
+function winningPair(pair1, pair2) {
+  if (!pairCheck(pair1) && !pairCheck(pair2)) return []
+  else if (pairCheck(pair1) && pairCheck(pair2)) {
+    const pair1Num = checkNumber(pair1)
+    const pair2Num = checkNumber(pair2)
+    return pair1Num[0] > pair2Num[0] ? pair1 : pair2
+  } else if (!pairCheck(pair1)) return pair2
+  else return pair1
+}
```

Comment:
You don't need to change this but going forward I'd like you to think about trying to find ways to avoid lengthy if / else / if chains like this, it adds a lot of room for strange bugs to happen